packages/dds/merge-tree/src/opBuilder.ts

```diff
@@ -88,11 +89,14 @@ export function createRemoveRangeOp(start: number, end: number): IMergeTreeRemov
  *
  * @internal
  */
-// eslint-disable-next-line import/no-deprecated
-export function createObliterateRangeOp(start: number, end: number): IMergeTreeObliterateMsg {
+export function createObliterateRangeOp(
```

Comment:
i don't think we should add support for sideness here yet, as we don't support sidedness yet, and it will all need to be gated behind a feature flag